feat: management command for triggering a status update to exam attempt

alangsto opened the pull request:
**Description:**

Add management command for updating an attempt status based on its associated review. 

**Pre-Merge Checklist:**

- [x] Updated the version number in `edx_proctoring/__init__.py` and `package.json` if these changes are to be released.
- [x] Described your changes in `CHANGELOG.rst`
- [x] Confirmed Github reports all automated tests/checks are passing.
- [ ] Approved by at least one additional reviewer.

**Post-Merge:**

- [ ] Create a tag matching the new version number.

ashultz0 commented on edx_proctoring/management/commands/tests/test_update_attempt_status_from_review.py:
```diff
@@ -0,0 +1,107 @@
+            student_id=self.user.id,
+        )
+
+        # update both reviews, and use update instead of save to avoid triggering a post_save signal
```
clever

MichaelRoytman commented:
Can we document somewhere the need for this management command, please? Most of the context is in an ephemeral Slack thread. Putting that context in the commit message or the MST ticket that you would link to from the commit message or the PR body would be helpful in the future for understanding why this management command was added.

alangsto commented:
> Can we document somewhere the need for this management command, please? Most of the context is in an ephemeral Slack thread. Putting that context in the commit message or the MST ticket that you would link to from the commit message or the PR body would be helpful in the future for understanding why this management command was added.

Just added a comment to the MST ticket and updated the commit message as well.

matthugs reviewed (approved):
I was worried about exam attempts that had been reset during the outage, but it seems like we'll use the newer history table for those so we shouldn't have trouble passing over them without trying to update a no-longer-existing attempt.
